template.tpl

```diff
@@ -318,6 +318,43 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "products",
+        "displayName": "Product Information",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Product ID",
+            "name": "id",
+            "type": "TEXT",
+            "valueHint": ""
```

Comment:
Do we wanna provide a hint that this ID should match the item's ID in the catalog?

Reply:
Good idea!